src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -211,8 +210,13 @@ public static function singularify($plural)
             }
         }
 
+        $irregularNouns = array(
```

Comment:
Inline arrays are can't benefit from some opcache optims until php7. A private static would fix that